google-marketplace/chart/datadog-mp/values.yaml

```
@@ -8,6 +8,7 @@ datadog-operator:
   serviceAccount:
     create: false
     name: placeholder
+  toolVersion: "google-marketplace"
```

```suggestion
  toolVersion: "google_marketplace"
```
nit: can we use an underscore? up for debate, but since it's "tag-like" that'd be my preference